README.md

```diff
@@ -1 +1,49 @@
-# lift_30_sbt
+
+Template Applications for Lift 3.0
+=======
+#Lift version 3.0 templates
+
+This repository contains templates for Lift projects.
+
+These templates offer a starting point for your Lift-based project.
+
+If you want to start with Scala 2.11.x, use the templates in the scala_211 directory
```

Comment:
@Shadowfiend thanks for the clarification, makes sense I will update the PR stay tuned.